ipykernel/kernelapp.py

```diff
@@ -149,11 +149,13 @@ def abs_connection_file(self):
         help="""ONLY USED ON WINDOWS
         Interrupt this process when the parent is signaled.
         """)
-
+    
     def init_crash_handler(self):
-        # Install minimal exception handling
-        sys.excepthook = FormattedTB(mode='Verbose', color_scheme='NoColor',
-                                     ostream=sys.__stdout__)
+        sys.excepthook = self.excepthook
+    
+    def excepthook(self, etype, evalue, tb):
+        # write uncaught traceback to 'real' stderr, not zmq-forwarder
+        traceback.print_exception(etype, evalue, tb, file=sys.__stderr__)
```

Comment:
Seems like a crash should go to stderr. I'll call the stdout destination a bug in addition to the formatting.